patches/chrome-installer-linux-debian-build.sh.patch

```diff
@@ -38,7 +38,17 @@ index b5755a2ce97534dbcd7d1ad8bd977bb12679f683..541f70f9ca69cb1ae01392ea1dfe5170
        grep '^ Depends: ' | sed 's/^ Depends: //' | sed 's/, /\n/g' | \
        LANG=C sort > "${ACTUAL_DEPENDS}"
    BAD_DIFF=0
-@@ -160,8 +160,11 @@ verify_channel() {
+@@ -125,6 +125,9 @@ do_package() {
+   fi
+   log_cmd fakeroot dpkg-deb ${COMPRESSION_OPTS} -b "${STAGEDIR}" .
+   verify_package "$DEPENDS"
++  cd "${STAGEDIR}/${INSTALLDIR}/"
```

Comment:
use `;` or `&&` to put this all on one line

Reply:
fixed